testsuite/kuadrant/authorino.py

```diff
@@ -133,9 +132,3 @@ def authorization_url(self):
     @property
     def oidc_url(self):
         return self._oidc_url
-
-    def commit(self):
```

Comment:
If you remove this here, remove it from PreexistingAuthorino as well

Reply:
This is removed from PreexistingAuthorino